client/src/components/AuditAdmin/AuditAdminView.tsx

```diff
@@ -48,16 +52,19 @@ const AuditAdminView: React.FC = () => {
         isDrawSampleComplete(rounds)
       ) {
         queryClient.invalidateQueries(jurisdictionsQueryKey(electionId))
+        queryClient.invalidateQueries(
+          jurisdictionsWithLastLoginQueryKey(electionId)
+        )
```

Comment:
I wasn't totally clear on this in my last comment, but we can remove all of the invalidations for `jurisdictionsWithLastLoginQueryKey` because they are covered by `jurisdictionsQueryKey`.

Reply:
Sorry that was clear. It just got lost in the other updates. I'll open a PR.